src/action/wallet.js

```diff
@@ -245,7 +246,7 @@ class WalletAction {
       this.initResetPassword();
       return this._notification.display({ msg: errorMsg });
     }
-    this._nav.goWait();
+    this._payment.initWaitScreen({ copy: 'Updating password...' });
```

Comment:
Why does the wallet need access to the payment action for this? Would probably be cleaner to either:

a. create a wait action
b. let actions call `nav.goWait({ copy: 'Updating password...' }) and set `store.waitScreenCopy` in the nav action

Considering it's only a one liner, I'd lean towards `b.`

Reply:
Just remembered this PR exists lol. Updating!

Reply:
Updated!